### PR TITLE
win_virtio_driver_update_test.cfg: add two cases for pvpanic

### PR DESCRIPTION
**Pull request by kimi1978wy**
add install/uninstall and upgrade/downgrade driver cases for pvpanic

Signed-off-by: Yu Wang <wyu@redhat.com>

id:1533751 1533752

### CODE REVIEW

**Comment by kimi1978wy** [comment]
(1/2) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_pvpanic.before_bg_test.uninstall_install: PASS (1844.41 s)
 (2/2) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_pvpanic.before_bg_test.downgrade_upgrade: PASS (1600.98 s)

**Comment by hereischen** [comment]
Acked by: Haotong Chen hachen@redhat.com

File: qemu/tests/cfg/win_virtio_driver_update_test.cfg [hunk 1]
```diff
@@ -131,3 +131,16 @@
...
                 read_rng_cmd = for /l %i in (1, 1, 1000) do "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
                 driver_check_cmd = ""
+        - with_pvpanic:
+            only before_bg_test
+            no Host_RHEL.m6
+            driver_name = pvpanic
+            device_name = "QEMU PVPanic Device"
+            run_bgstress = pvpanic
+            event_check = "GUEST_PANICKED"
+            disable_shutdown = yes
```
Comment (xiagao) [inline qemu/tests/cfg/win_virtio_driver_update_test.cfg]: no disable_shutdown

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**